testsuite/integration-arquillian/tests/base/src/test/resources/adapter-test/keycloak-saml/ecp-sp/WEB-INF/keycloak-saml.xml

```diff
@@ -18,7 +18,7 @@
 <keycloak-saml-adapter xmlns="urn:keycloak:saml:adapter"
                        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                        xsi:schemaLocation="urn:keycloak:saml:adapter http://www.keycloak.org/schema/keycloak_saml_adapter_1_7.xsd">
-    <SP entityID="http://localhost:8081/ecp-sp/"
+    <SP entityID="saml-ecp-sp"
```

Comment:
entityIds has to be URLs. See https://docs.oasis-open.org/security/saml/v2.0/saml-core-2.0-os.pdf, line 3315

Reply:
@hmlnarik thank you for pointing that out, I wasn't aware of this. The PR should be updated accordingly now.